sdk/src/client/node_api/core/routes.rs

```diff
@@ -301,13 +293,13 @@ impl ClientInner {
 
     /// Get the metadata for a given `OutputId` (TransactionId + output_index).
     /// GET /api/core/v2/outputs/{outputId}/metadata
-    pub async fn get_output_metadata(&self, output_id: &OutputId) -> Result<OutputMetadataDto> {
+    pub async fn get_output_metadata(&self, output_id: &OutputId) -> Result<OutputMetadata> {
```

Comment:
This fails now with `Error: Node(Reqwest(reqwest::Error { kind: Decode, source: Error("missing field `outputId`", line: 1, column: 300) }))` because in the response the outputId is split into transactionId and outputIndex